### PR TITLE
feat: settings for mg_amount_selector experiment were removed

### PR DESCRIPTION
**Pull request by roger-in-kiva**

### CODE REVIEW

**Comment by mcstover** [comment]
@roger-in-kiva DOesn't the `landing-form-experiment` component usage also need to be removed? I thought that had been deleted.

**Comment by roger-in-kiva** [comment]
> @roger-in-kiva DOesn't the `landing-form-experiment` component usage also need to be removed? I thought that had been deleted.

The component is not more in the repo, part of the clean up was here, so, just the updated files were needed to fix.
@mcstover

**Comment by mcstover** [comment]
@roger-in-kiva It looks like `landing-form-experiment` is still present in this file on line 112

File: src/pages/MonthlyGood/MonthlyGoodLandingPage.vue [hunk 1]
```diff
@@ -110,13 +109,6 @@
 				:button-text="heroPrimaryCtaText"
 				v-if="!isMonthlyGoodSubscriber && !isExperimentActive && !hasModernSub"
```
Comment (mcstover) [inline src/pages/MonthlyGood/MonthlyGoodLandingPage.vue:110]: `isExperimentActive` check should be removed too

Comment (mcstover) [reply src/pages/MonthlyGood/MonthlyGoodLandingPage.vue:110]: I think this prob still works b/c `undefined` matches this check though ;)

**Review by mcstover — APPROVED** [review]
One more small item in the comments

**Review by mcstover — APPROVED** [review]

**Comment by mcstover** [comment]
Not sure why coveralls was delayed so long. Merging.

**Pull request merged**